org.tryton.Tryton-68.yml

```diff
@@ -0,0 +1,37 @@
+id: org.tryton.Tryton-68
+sdk: org.gnome.Sdk
+runtime: org.gnome.Platform
+runtime-version: '44'
+command: tryton
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=all
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
```

Comment:
if you use GNotification, you shouldn't need this dbus permission

Reply:
It is using https://docs.gtk.org/gio/method.Application.send_notification.html